--- /usr/bin/plantuml	2020-03-10 18:57:51.000000000 +0100
+++ bin/plantuml	2024-02-29 13:24:57.106063939 +0100
@@ -29,4 +29,10 @@
 HEADLESS=
 [ -z "$DISPLAY" ] && HEADLESS=-Djava.awt.headless=true
 
-$JAVA -Djava.net.useSystemProxies=true $HEADLESS -jar /usr/share/plantuml/plantuml.jar "$@"
+## 2024-02-29: Stépahne ALBERT: added `PLANTUML_ROOTDIR` support.
+if [ -z "${PLANTUML_ROOTDIR}" ]
+then
+    PLANTUML_ROOTDIR="${HOME}/opt/local/share"
+fi
+echo "PLANTUML_ROOTDIR=\"${PLANTUML_ROOTDIR}\""
+$JAVA -Djava.net.useSystemProxies=true $HEADLESS -jar ${PLANTUML_ROOTDIR}/plantuml/plantuml.jar "$@"
